src-java/testing/functional-tests/src/test/groovy/org/openkilda/functionaltests/spec/flows/ConnectedDevicesSpec.groovy

```diff
@@ -509,7 +509,10 @@ srcDevices=#newSrcEnabled, dstDevices=#newDstEnabled"() {
     def "System properly detects devices if feature is 'off' on switch level and 'on' on flow level"() {
         given: "A switch with devices feature turned off"
         assumeTrue(topology.activeTraffGens.size() > 0, "Require at least 1 switch with connected traffgen")
-        def tg = topology.activeTraffGens.shuffled().first()
+        // Skip tg88 due to the issue https://github.com/telstra/open-kilda/issues/5663
+        def tgsWhere5663IsNotReproduced = topology.activeTraffGens
```

Comment:
The root cause of issue 5663 has been found, there is no need to skip tg88. The appropriate changes will be added in the scope of regular flow refactoring.